themes/default/content/docs/guides/continuous-delivery/create-new-cicd-integration.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Other
+title: Create New CI/CD Integration
```

Comment:
This title might be a bit misleading since you are not actually "creating" a new CI/CD integration. This doc basically describes an escape-hatch mechanism for a user to set some custom env vars for Pulumi to associate an update with in case their CI/CD is not automatically picked up by the Pulumi CLI. I do like the idea of having a different title than "Other" but just that "Create New CI/CD Integration" means something else to me.

Reply:
@praneetloke do you have any suggestions for other titles that might work better?

Reply:
My possibly awful suggestion is "Custom CI/CD Detection"? 😁

Reply:
@praneetloke okay i read the page, instead of just the intro paragraph which is where i got the create language.. heh. ive updated that, and also tried out the name `Add Support for CI Systems`. what do you think? still misleading? if so, i will use your suggestion directly, though i think someone should maybe update the first couple paragraphs to match that.

Reply:
Yeah sounds good. I'll connect with you tomorrow. I think we should rephrase the intro as you suggest.